download.py

```diff
@@ -34,7 +34,7 @@ def download_run_fasterq_dump(accession, term, total_spots, out):
     """
     download_bash_command = f"fasterq-dump {accession} -O {out}/{term} -p"
     logging.debug(download_bash_command)
-    logging.info('Try to download {} file'.format(accession))
+    logging.info('Try to download %s file', accession)
```

Comment:
It's cool that you remember that trick with logging formatting (I completely forgot already).